modin/pandas/accessor.py

```diff
@@ -236,6 +236,21 @@ def to_ray_dataset(self):
         """
         return to_ray_dataset(self._data)
 
+    def to_dask_dataframe(self):
```

Comment:
It seems we should have both to_dask_dataframe and to_dask_series for a Modin df and series, respectively.

Reply:
Since there is a direct mapping dask series <-> modin series and dask df <-> modin df, I think it makes sense to just leave to_dask.

Reply:
Dask is a flexible parallel computing library for analytics. If you want to process tabular data by parallelizing pandas, you need to use Dask DataFrame. Since you previously required the function to be called "to_ray_dataset", this function must be called "to_dask_dataframe"!

Reply:
> Dask is a flexible parallel computing library for analytics. If you want to process tabular data by parallelizing pandas, you need to use Dask DataFrame. Since you previously required the function to be called "to_ray_dataset", this function must be called "to_dask_dataframe"!

Ok for me, but we also have to add `to_dask_series` in that case.

Reply:
Guys, I am now thinking that we should have named `to_ray_dataset` as just `to_ray`. Here I think we should name the function just `to_dask` and convert a Modin df to a dask df and a Modin series to a dask series. Just came to my mind that we overcomplicate things. `to_ray_dataset`, `to_ray_something_else`, `to_dask_df`, `to_dask_series`, `to_dask_array`, etc. We are adding a lot of stuff like that we have to support. Having just `to_ray` and `to_dask` seem sufficient since conversion to other ray objects or dask objects should be implemeneted by those libraries and they support that. E.g, dask_s.to_frame(), dask_df.to_dask_array(), etc. What do you think?

Reply:
You moved these function to Modin API and it is inherited by both Dataframe and Series. How do you suggest to split them?
Also we have the different implementation for `to_pandas` function. In that case we have the union way to cast our DataFrame/Series to pandas DataFrame/Series respectively.

Reply:
> Guys, I am now thinking that we should have named `to_ray_dataset` as just `to_ray`. Here I think we should name the function just `to_dask` and convert a Modin df to a dask df and a Modin series to a dask series. Just came to my mind that we overcomplicate things. `to_ray_dataset`, `to_ray_something_else`, `to_dask_df`, `to_dask_series`, `to_dask_array`, etc. We are adding a lot of stuff like that we have to support. Having just `to_ray` and `to_dask` seem sufficient since conversion to other ray objects or dask objects should be implemeneted by those libraries and they support that. E.g, dask_s.to_frame(), dask_df.to_dask_array(), etc. What do you think?

Yes, this is exactly what I wanted to explain before. I think this is a more suitable option. But we have already released the `to_ray_dataset` function. So I think now we will have more difficulties with renaming it (mark it as deprecated and remove later), if we choose this option.

Reply:
> You moved these function to Modin API and it is inherited by both Dataframe and Series. How do you suggest to split them? Also we have the different implementation for `to_pandas` function. In that case we have the union way to cast our DataFrame/Series to pandas DataFrame/Series respectively.

I don't see a good way to do that, that's why I suggest to implement only `to_dask`.

Reply:
BTW, I look at Ray dataset as a component of the framework, and not as a specific type. The problem that arose here is that the function name uses a specific type (DataFrame or Series). I still don't mind calling to_dask_[name of the component in Dask].

Reply:
@anmyachev, I would prefer to rename `to_ray_dataset` to `to_ray` by analogy with `to_dask`, `to_pandas`, etc. and deprecate `to_ray_dataset`. Why do you think we shouldn't rename `to_ray_dataset`?

Reply:
> I look at Ray dataset as a component of the framework, and not as a specific type

We can choose any type of the data format of a library and convert a Modin object to it. All other conversions to a third party library object should be (is) implemented by that library. No need for us to support that stuff.

Reply:
> BTW, I look at Ray dataset as a component of the framework, and not as a specific type. The problem that arose here is that the function name uses a specific type (DataFrame or Series). I still don't mind calling to_dask_[name of the component in Dask].

The fact is that "dataset" is not component of the Ray, but "dataframe" is a component of the Dask.
In code it seems like this:
```python
import ray.data as rd
ray_df : rd.Dataset = rd.from_pandas(...)

import dask.dataframe as dd
dask_df : dd.DataFrame = dd.from_delayed(...)
```

Reply:
I assume you see a one-to-one correspondence between Python modules and library components?

Reply:
@YarShev @Retribution98 If you're on the same side on this issue, feel free to continue with this decision.